### ISSUE TITLE
Q: How to produce a breakdown of how much it took to build each dependency of a target (recursively)?

### ISSUE THREAD

**Issue by andportnoy** [bazelbuild/bazel#24572]
### Page link:

https://bazel.build/advanced/performance/build-performance-metrics#execution-graph-log

### Problem description (include actual vs expected text, if applicable):

I'd like to be able to analyze build performance in terms of cumulative CPU time spent building all targets that are dependencies of the target I'm building. How can I do this?

**Comment by meisterT** [comment]
Are you actually interested in CPU time and not wall time? What's the background?

**Comment by andportnoy** [comment]
Yes, CPU time. I'm trying to understand where time is spent in my build to guide optimization and see if I can exclude some expensive subtargets if they aren't really needed.
